dev/com.ibm.ws.kernel.boot.ws-server/publish/bin/server.bat

```diff
@@ -341,15 +341,17 @@ goto:eof
 goto:eof
 
 :package
-  call:serverEnv
+  call:serverEnvAndJVMOptions
+  if not %RC% == 0 goto:eof
   call:serverExists true
   if %RC% == 2 goto:eof
 
-  !JAVA_CMD_QUOTED! !JAVA_PARAMS_QUOTED! --batch-file=--package !PARAMS_QUOTED!
+  !JAVA_CMD_QUOTED! !JAVA_AGENT_QUOTED! !JVM_OPTIONS! !JAVA_PARAMS_QUOTED! --batch-file=--package !PARAMS_QUOTED!
```

Comment:
Is JAVA_AGENT really needed here? I would think that would only be necessary for a server

Reply:
Whenever I run with Java 8 (without the JAVA_AGENT_QUOTED in the cmd), I get [WARNING ] TRAS0032W: Runtime injection of detailed method trace is not available. Adding the JAVA_AGENT_QUOTED seems to resolve the issue.